client-mvp-04/src/components/createAccount/createAccountContainer.test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
```

Comment:
Your component doesn't connect to redux store. These imports are redundant. Please, remove 2-3 lines.